Examples/Gdip.Tutorial.11-Using.BRAs.to.perform.animations.ahk

```diff
@@ -0,0 +1,350 @@
+; gdi+ ahk tutorial 11 written by tic (Tariq Porter)
+; Requires Gdip.ahk either in your Lib folder as standard library or using #Include
+;
+; Tutorial to demonstrate how to use BRA files, in this instance for animation
+
+; #SingleInstance, Force
+; #NoEnv
+
+; Uncomment if Gdip.ahk is not in your standard library
+;#Include, Gdip.ahk
+
+; Start gdi+
+If !pToken := Gdip_Startup()
+{
+	MsgBox, 48, gdiplus error!, Gdiplus failed to start. Please ensure you have gdiplus on your system
+	ExitApp
+}
+; On exiting the program we will go to Exit subroutine to clean up any resources
+; OnExit, Exit
+OnExit("AppExit")  ; Requires [v1.1.20+]
+
+; I've added a simple new function here, just to ensure if anyone is having any problems then to make sure they are using the correct library version
+if (Gdip_LibraryVersion() < 1.30)
```

Comment:
Can you also bump the `Gdip_LibrarySubVersion()` number in the Gdip_All file, and also bump the version and date and comments at the top of the Gdip_All file?